blueocean-web/src/main/js/try.js

```diff
@@ -0,0 +1,18 @@
+var $ = require('jquery-detached').getJQuery();
+var jsModules = require('@jenkins-cd/js-modules');
+
+$(document).ready(() => {
+    var tryBlueOcean = $('<div class="try-blueocean header-callout">Try Blue Ocean UI ...</div>');
```

Comment:
No, just need CSS for the positioning:

`.bo-callout { position: fixed; z-index: 2000; left: 50%; margin-left: -8em }` <-- guesses

I mean 7 files instead of 2, downloads to the browser, etc... it *is* a lot of code when you just need a link...